app/views/renewal/customers_outside_uk.scala.html

```diff
@@ -59,30 +60,28 @@
                 inline = true
             )
         }
+    }
 
-        @panel(
-            field = Some(f(s"$target")),
-            jsHidden = false,
-            data = Seq("data-toggle-new")
+    @fieldset(
+        f = f("countries-fieldset"),
+        legend = "renewal.customers.outside.uk.hint.lbl",
+        classes = Seq("countries-fieldset")
+    ) {
```

Comment:
just to confirm... hint is not necessary anymore?

Reply:
My error completely. Good spot.